exporter/tests/test_forms.py

```diff
@@ -0,0 +1,84 @@
+from unittest import mock
+
+from django.test import TestCase
+
+from pages.factories import ConditionPageFactory, ConditionsPageFactory
+from pages.models import EditorialPage
+
+from ..forms import ExportForm
+
+
+class ExportFormTestCase(TestCase):
+    def setUp(self):
+        """
+        Creates:
+
+        root
+            homepage
+                conditions (live)
+                    level1-a (live)
+                    level1-b (draft)
+                    level1-c (live)
+                        level2-c (live)
+        """
+        super().setUp()
```

Comment:
No, it's not necessary. 
I tend to do this in case I want to add a mixin or a superclass later on without the worry of forgetting to add the super bit.

Can delete it if you want.